staging/prometheus-operator/Chart.yaml

```diff
@@ -12,7 +12,7 @@ sources:
   - https://github.com/coreos/kube-prometheus
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 8.7.4
+version: 8.8.4
```

Comment:
shouldn't this be `8.7..5` since it's a fix or would you consider this a new feature? In that case should it be `8.8.0`?

Reply:
Oh, I am making a breaking change since one field, the serviceURL is no longer valid. It is being deprecated and not consumed anywhere atm.